playground/infrastructure/helper.py

```diff
@@ -13,14 +13,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import re
 import os
+import yaml
 
+from dataclasses import dataclass
 from typing import List
-from api.v1.api_pb2 import Sdk, SDK_UNSPECIFIED, SDK_JAVA, Status, STATUS_UNSPECIFIED
+from api.v1.api_pb2 import Sdk, SDK_JAVA, SDK_UNSPECIFIED, STATUS_UNSPECIFIED, SDK_GO, SDK_PYTHON, SDK_SCIO
 
-SUPPORTED_SDK = {'java': SDK_JAVA}
-PATTERN = re.compile('Beam-playground:\n {2} *name: \w+\n {2} *description: .+\n {2} *multifile: (true|false)\n {2} *categories:\n( {4} *- [\w\-]+\n)+')
+SUPPORTED_SDK = {'java': SDK_JAVA, 'go': SDK_GO, 'py': SDK_PYTHON, "scala": SDK_SCIO}
```

Comment:
Let's remove Scala from here for now. We will support later than Go and Python.

Reply:
Changed.